doc/source/releases/25.0_announce_user.rst

```diff
@@ -0,0 +1,563 @@
+  accessing datatypes/permissions and more — is now organized into clear, intuitive tabs.
+- **Improved usability**. This update provides a one stop shop for dataset management,
+  reducing clutter and confusion from multiple buttons and options.
+
+This change enhances consistency across the Galaxy interface and offers a more intuitive experience
+for both new and experienced users working with datasets.
+[`#20154 <https://github.com/galaxyproject/galaxy/pull/20154>`__]
+
+.. raw:: html
+
+   <iframe width="100%" height="360" style="margin-bottom: 1em;" src="https://www.youtube.com/embed/vQ8pSgiVTpQ?si=Ab61hzSs94WswoIR" title="Galaxy 25.0 - The New Unified Dataset View" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+Allow Rerunning Workflows with the Same Inputs and Parameters
+--------
+
+Users can now easily rerun any workflow with the exact same inputs and parameters as a previous run - 
+with a simple click of the new "Rerun" button in the workflow invocation view. This feature streamlines
+reproducibility and troubleshooting by letting you repeat analyses with a single click. The workflow run
+form also detects and notifies of any changes to inputs compared to the original run!
+[`#20032 <https://github.com/galaxyproject/galaxy/pull/20032>`__]
+
+.. raw:: html
+
+   <iframe width="100%" height="360" style="margin-bottom: 1em;" src="https://www.youtube.com/embed/CaWkstTMXJ0?si=ntvZHNjZ7GuhWmp8" title="Galaxy 25.0 - Rerun Workflows" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+An Enhanced Workflow Run Form Interface
+--------
+
+Galaxy 25.0 introduces a streamlined workflow run interface that allows users to upload inputs, create
+collections, and configure parameters — all within a single, unified form.
+
+- **Integrated data upload and collection creation**. Users can now upload datasets or build
+  collections directly from the workflow run form, without navigating away or switching contexts.
+- **Visual input indicators**. Each input now displays a visual status indicator, making it easy to see
+  which inputs are ready and which still need attention.
+- **Improved settings panel**. The run form includes a clearer, more organized settings menu with
+  well-annotated options for configuring the workflow execution.
+- **Live-updating workflow graph**. A dynamic graph of the workflow appears alongside the form,
+  updating in real time as users populate inputs — helping them understand the structure and flow
+  of the analysis.
+
+These improvements were inspired by users encountering curated workflows from platforms like the IWC
+Browser or BRC Analytics, and are designed to reduce friction, improve clarity, and enable users to
```

Comment:
Should we make both IWC and BRC Analytics links?

Reply:
Yes I like that